EDI/Overview/03_TipsTricks.md

```diff
@@ -0,0 +1,74 @@
+# Tips and tricks for preparation
+
+
+
+## Introduction
+
+UN/EDIFACT (in the following short EDIFACT) is a cross-industry international standard for the format of electronic data in business transactions. EDIFACT is one of several international EDI standards. A UN organization called CEFACT, which is affiliated to the UNECE, is responsible for the EDIFACT standard.
+[^1]    
+
+The EDIFACT standard defines a scheme according to which messages should be structured. It is used for data exchange and transfer between business partners, for example invoice or order data. The standard defines a list of messages such as ORDERS, INVRPT, INVOIC, DELFOR, etc. which are used for specific purposes. For example, an INVRPT represents an inventory report.   
+
+The EDIFACT standard is available in different versions. Every year one or two versions are released (often referred to as an a or b version). Your business partners, the receivers of the EDIFACT messages, define their specifications according to a specific version. Since the *XML/EDIFACT & UN/EDIFACT Exporter* plugin provides the structure for all versions, you can find the particular version specified by your business partner in the *DataHub Exporter* module.   
+
+Note that the structure of the format is standardized, not the usage. As a result, each user/business partner defines individually the data to be provided for the message structure. This also applies to the classification of the segments and fields (mandatory, required, optional, and others), which may sometimes differ from the EDIFACT standard.   
+>[INFO] In the following, when we refer to a user or a business partner, we mean the company to which you need to send the messages to, such as your supplier, your customer, or your fulfillment partner.
+
+
+
+## Tips for preparation in advance
+
+- Many internet providers offer EDIFACT information. For a first impression, we can recommend the https://www.truugo.com/edifact/ website. It provides a well-structured presentation of the EDIFACT messages in their different versions.  
+
+- Ask your business partner for the EDIFACT message specification. Sometimes it is also called message implementation guide. Specifications from some major vendors are also available for download on the Internet.
+
+- Ask your business partner for the connection details that specify to which address and with which credentials the data are to be transferred. For example, get to know both the GLN number (Global location number) of your business partner and the GLN number of your company.
+
+- Review this specification for the segments your business partner has defined as required and ensure that you have this data available.   
+
+- Note the following: 
+    - Some segments are mandatory (M) only in case if the optional segment (O) above is to be filled. See the example:
```

Comment:
Ich finde den Satz schwer zu verstehen. Sollte es nicht "only in case the optional segment..." (oder if the optional segment) sein?